docs/source/peers/peers.md

```diff
@@ -254,101 +256,94 @@ form a blockchain network, it's worth spending a few moments understanding how
-application has received a sufficient number of signed proposal responses,
-the first phase of the transaction flow is complete. Let's examine this phase in
-a little more detail.
+We've seen that peers form the basis for a Fabric blockchain network, hosting
+ledgers and smart contracts which can be queried and updated by peer-connected
+applications. The mechanism by which applications and peers interact with each
+other to keep the ledger current and consistent across a channel --- the Fabric
+Gateway service --- is mediated by special nodes called *orderers*. It's to
+these orderer nodes that we now turn our attention.
+
+An ledger update transaction is different from a query transaction because a single
+peer cannot, on its own, update the ledger --- that requires the consent of other
+peers in the network, a process known as *consensus*. When the peers required to
+approve the transaction do so, and the transaction is committed to the ledger,
+the gateway service notifies the applications that the ledger has been updated.
+Peers and orderers work together to manage consensus.
+
+### Orderers and the Three Phases
+
+Previously, in the [Applications and Peers](#peers-and-application) topic, we
+saw how a client application that requests a ledger update initiates a three-phase
+process which, with help from the Fabric Gateway service, ensures that
+all peers in a Fabric network maintain current and consistent copies of the ledger.
+As described, orderers are also involved; the following sections take a closer
+look at the three-phase process from the perspective of ordering nodes and peers
+(i.e. the ordering service and the gateway service).
+
+### Phase 1: Transaction Proposal
+
+Phase 1 of the transaction workflow involves an interaction between a client
+application, peers and orderers. In Phase 1, the client application initiates
+a request to the Fabric Gateway service to evaluate a transaction proposal.
+
+The target peer, selected by the client application, executes the transaction
+by invoking chaincode --- this step can be described as simulating the transaction,
+because it runs the transaction without any effect on the ledger. The peer then
+returns its transaction result to the client.  
+
+The gateway service also forwards the transaction proposal to the required
+*endorsing peers* (based on the endorsement policies), which also execute the
+transaction and return their results to the peer. The gateway service collects
+all responses, and if they collectively satisfy the endorsement policies, forwards
+the transaction to the ordering service.
+
+If the endorsement policies are not satisfied, or if the proposed transaction would
+write to a private data collection, (as *transient* data) the gateway service returns
+a failed proposal message to the client. The client application must then a) be updated
+to explicitly specify the organizations required for endorsement and b) resubmit the
+transaction proposal.
+
```

Comment:
Again, the client won't resubmit it.
The client will be written to either delegate the endorsement choice to the gateway service, or will be written to specify endorsing peers. It will never be written to do some hybrid approach with automatic resubmit.

Reply:
Calling with @andrew-coleman to avoid more cycling on GitHub PRs.